test/parachain/runtime/build.rs

```diff
@@ -19,7 +19,7 @@ use wasm_builder_runner::WasmBuilder;
 fn main() {
 	WasmBuilder::new()
 		.with_current_project()
-		.with_wasm_builder_from_crates("1.0.9")
+		.with_wasm_builder_from_git("https://github.com/paritytech/substrate.git", "0b30207969fdde85e0dad785750757399cd0e3e4")
```

Comment:
There is no branch or tag we can use instead? Like cumulus-branch for instance?

Reply:
A commit is a tag.

Reply:
And why didn't you pin to the branch instead? You needed that commit in particular? It wouldn't have work on cumulus-branch?